tests/target/issue_6059.rs

```diff
@@ -0,0 +1,3 @@
+fn float_range() {
+    let _range = 3. / 2. ..4.;
```

Comment:
Thanks for your help on this!

What happens if the `2.` was written as `2.0`? Do we still add the space? Can you include a test case for that.

Reply:
No worries! If the literal ends with a dot it doesn't add a space, I think that's intended right? I added a test case for it